docs/docs/guides/metadata.md

```diff
@@ -0,0 +1,200 @@
+## NFT Ticket Metadata Example
+
+```json
+{
+  "factory_id": "16247fb7-1a2c-4899-a581-c46b319ef600",
+  "release_type": "ticket",
+  "name": "Hotdog eating contest",
+  "description": "How many dogs can you eat? Come join in on the fun at The Dog House!",
+  "venue": "The Dog House",
+  "address": "123 Cactus Canyon",
+  "start": "1652276545",
+  "end": "1652362945"
+}
+```
+
+## Subgraph
+
+```json
+{
+  "id": "0x6813c7e729841546a39d38897b4c75f410b01f1d",
+  "factory_id": "16247fb7-1a2c-4899-a581-c46b319ef600",
+  "release_type": "ticket",
+  "properties": [
+    {
+      "key": "name",
+      "value": "Hotdog eating contest"
+    },
+    {
+      "key": "description",
+      "value": "How many dogs can you eat? Come join in on the fun at The Dog House!"
+    },
+    {
+      "key": "venue",
+      "value": "The Dog House"
+    },
+    {
+      "key": "date",
+      "value": "2022-04-29"
+    },
+    {
+      "key": "address",
+      "value": "123 Cactus Canyon"
+    },
+    {
+      "key": "start",
+      "value": "21:38"
+    },
+    {
+      "key": "end",
+      "value": "23:38"
+    }
+  ]
+}
+```
+
+You will see that the [subgraph](/glossary#subgraph) has converted the key/value pairs in the JSON blob into a list of properties. Then if you want to search for all events where the venue is "The Dog House", you can perform this query:
+
+```graphql
+{
+  properties(where: { value_contains: "The Dog House" }) {
+    tokens {
+      id
+      factory_id
+    }
+  }
+}
+```
+
+## Release Types
+
+The release types help developers when filtering and sorting the [subgraph](/glossary#subgraph). We highly recommend you choose one of the release types below. If you would like us to add a release type to this list, please let us know in [Discord](https://discord.com/invite/8WV52tVqbZ).
+
+- `video`
+- `audio`
+- `ticket`
+- `image`
+
+## Factory ID
+
+A factory ID is a unique identifier that's used for filtering the [subgraph](/glossary#subgraph) to only show NFTs created in specific factory frontend. This can be any string value, but for uniqueness, we recommend using [UUIDv4](https://www.uuidgenerator.net/). Simply add a key/value pair to your JSON blob in the metadata:
+
+```tsx
+{
+  "factory_id": "16247fb7-1a2c-4899-a581-c46b319ef600",
+  ...metadata
+}
+```
+
+Our [subgraph](/glossary#subgraph) will look for this `factory_id` and store it against the created `Token` entity.
+
+## Formatting
+
+### Date & Times
+
+We recommend any dates and times are formatted as a unix timestamp.
+
+```tsx
+{
+  "start": "1652276545", // start date/time of event
+  "end": "1652362945" // end date/time of event
+   ...metadata
+}
+```
+
+## Querying the Subgraph
+
+Each NFT is represented as a `Token` entity in the [subgraph](/glossary#subgraph).
+
+### Querying all tokens
+
+```graphql
+{
+  tokens {
+    id
+    factory_id
+    release_type
+  }
+}
+```
+
+### Querying a single token
+
+```graphql
+{
+  token(id: "0xc922b16f4e9d299fd5fc5b8375928fa761484042") {
```

Comment:
Is it worth highlighting here that the subgraph explicitly wants addresses formatted in lower case??? I can say from personal experience I have lost a fair bit of time trying to understand what is was about my query that was incorrect before looking at the casing.

Reply:
Yes, I will write a ticket for it.